xml/System.Threading/CancellationToken.xml

```diff
@@ -337,7 +337,7 @@
 ## Remarks  
  The cancellation token returned by this property cannot be canceled; that is, its <xref:System.Threading.CancellationToken.CanBeCanceled%2A> property is `false`.  
   
- You can also use the C#  [default(CancellationToken)](~/docs/csharp/language-reference/keywords/default.md) statement to create an empty cancellation token.  
+ You can also use the C#  [default(CancellationToken)](~/docs/csharp/language-reference/keywords/default.md) statement to create an empty (and equivalent) cancellation token. Two empty cancellation tokens are always equal.
```

Comment:
Could you make the following changes:
1. Please remove "(and equivalent)" from the first sentence. It's unclear what equivalent refers to in the context of the sentence.
2. Please move the second sentence to be an independent paragraph that follows the first paragraph. The second sentence is related to the equality of CancellationTokens, not to the C# default keyword.